themes/bootstrap5/templates/RecordTab/holdingsils/standard.phtml

```diff
@@ -66,6 +83,11 @@
               <span class="icon-link__label"><?=$this->transEsc($check ? 'Check Recall' : 'Recall This')?></span>
             </a>
           <?php endif; ?>
+          <?php if ($getThisEnabled && !$isOnlineResource): ?>
+            <a data-lightbox href="<?= $getThisURI ?>" rel="nofollow">
+              <?=$this->transEsc('Get this')?>
```

Comment:
We seem to lack translations for "Get this" -- also note that in this file, each instance of the phrase is capitalized differently. We have "Get This" here and "Get this" above. We should pick one and use it consistently.